compiler/testSuite/12-nested/order-v.bal

```diff
@@ -11,8 +11,7 @@ public function main() {
     EmptyTuple e2 = [];
     io:println(e1 < e2); // @output false
 
-
     boolean[0] b0 = [];
     float[0] f0 = [];
```

Comment:
Fixed length arrays not part of the subset yet. They will get added with tuples in subset14.